src/main/java/ua/goit/timonov/enterprise/module_6_2/view/menus/MainMenu.java

```diff
@@ -0,0 +1,85 @@
+package ua.goit.timonov.enterprise.module_6_2.view.menus;
+
+/**
+ * Main console menu
+ */
+public class MainMenu extends ConsoleMenu {
+
+    /* console menus for different groups of tasks */
+    private EmployeesConsoleMenu employeesConsoleMenu;
+    private DishesConsoleMenu dishesConsoleMenu;
+    private MenusConsoleMenu menusConsoleMenu;
+    private OrdersConsoleMenu ordersConsoleMenu;
+    private CookedDishConsoleMenu cookedDishConsoleMenu;
+    private StorageConsoleMenu storageConsoleMenu;
+
+    public void setEmployeesConsoleMenu(EmployeesConsoleMenu employeesConsoleMenu) {
```

Comment:
Don`t you think ConsoleMenu should be Spring-agnostic? (It shouldn`t depend on Spring)